ccd-client/src/main/java/uk/gov/hmcts/cmc/ccd/client/StartCaseApi.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.cmc.ccd.client;
+
+import org.springframework.cloud.netflix.feign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import uk.gov.hmcts.cmc.ccd.client.model.StartEventResponse;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+
+@FeignClient(name = "core-case-data-api", url = "${core_case_data.api.url}")
+public interface StartCaseApi {
+
+    @RequestMapping(
+        method = RequestMethod.GET,
+        value = "/caseworkers/{userId}/jurisdictions/{jurisdictionId}/case-types/{caseType}/event-triggers/{eventId}/token",
```

Comment:
shouldn't we use the citizen api?

Reply:
no, ccd does not allow solicitor role to submit a case with citizen endpoints.